src/settingsLoader.js

```diff
@@ -1,12 +1,14 @@
+var path = require('path');
+
 var DEFAULT_PORT = 3000
 exports.DEFAULT_PORT = DEFAULT_PORT
 var fs = require('fs');
 
-var DEFAULT_DOWNLOAD_PATH = process.cwd() + '\\downloads\\'
+var DEFAULT_DOWNLOAD_PATH = path.win32.join(path.normalize(process.cwd()),'\\settings.json')
```

Comment:
This is definitely wrong. 

On MacOS X:

```
> var path = require('path');
undefined
> path.win32.join(process.cwd(),'\\settings.json')
'\\Users\\rngadam\\EIC\\src\\ansible-setup\\settings.json
```

there's no way using win32 is correct. we're writing unix

What you want to do is this:

```
> path.join(process.cwd(), 'settings.json')
'/Users/rngadam/EIC/src/ansible-setup/settings.json'
```

https://nodejs.org/api/path.html#path_path_join_paths